src/Core/Migration/Migration1757598733AddMigrationFixesTable.php

```
@@ -17,13 +17,13 @@
 #[Package('fundamentals@after-sales')]
 class Migration1757598733AddMigrationFixesTable extends MigrationStep
 {
-    public const MIGRATION_FIXES_TABLE = 'swag_migration_fixes';
+    public const MIGRATION_FIXES_TABLE = 'swag_migration_fix';
 
     public const FIELDS = [
         'id' => 'BINARY(16) NOT NULL',
         'connection_id' => 'BINARY(16) NOT NULL',
         'main_mapping_id' => 'BINARY(16) NOT NULL',
-        'value' => 'JSON NOT NULL',
+        'value' => 'LONGTEXT NOT NULL',
```

As i mentioned, the JsonField of shopware does only accept an array. We have different value types

---

Considering that the getter and setter of the entity always expects to read / write JSON, I think it would still be better to enforce that on a DB level and keep the `json` type here. Otherwise the admin api might write some non json value into the DB and then PHP tries to read and fail suddenly. It would be better to fail early (during write) than late (during read).

Even better would be going with a custom DAL field type with it's own serializer that validates the data

---

see also my other comment here: https://github.com/shopware/SwagMigrationAssistant/pull/76#discussion_r2455390732